### ISSUE TITLE
Opening the same picture multiple times

### ISSUE THREAD

**Issue by DatJulian** [d2phap/ImageGlass#354]
I think it is very annoying, that when you open the same picture multiple times, it always creates a new window instead of focusing the one where it is already opened.
I would suggest an option to disable this, because maybe someone relies on this "feature" ;)

**Comment by d2phap** [comment]
You can uncheck the option `Allow multiple instances of the program` :)

**Comment by DatJulian** [comment]
Yes, i know that, but i want multiple instances EXCEPT when i already have one with that picture already open...

**Comment by smaragdus** [comment]
This is none of my business but this is the most retarded Issue I have ever seen, Either disable multiple instances or just don't open one and the same image multiple times.

**Comment by DatJulian** [comment]
Oh thanks for being so polite. It was just a _suggestion_, if you don't like it you can tell me so in a normal tone without insulting me.

**Comment by smaragdus** [comment]
@DatJulian 
I do not intend to insult anyone. This was my opinion. I am not used to censor myself.
I have tested heaps of image viewers and yet I have not seen behaviour similar to the one you suggested. If the developer decides to implement it I cannot object of course. As I mentioned above, this is none of my business.

**Comment by d2phap** [comment]
Added this to feature request!
Pro tip: you can use 👍 to vote the post.
